src/main/java/org/sonarlint/intellij/config/project/SonarLintProjectAnalyzersPanel.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * SonarLint for IntelliJ IDEA
+ * Copyright (C) 2015 SonarSource
+ * sonarlint@sonarsource.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
+ */
+package org.sonarlint.intellij.config.project;
+
+import com.intellij.openapi.diagnostic.Logger;
+import com.intellij.openapi.project.Project;
+import com.intellij.ui.ToolbarDecorator;
+import com.intellij.ui.table.JBTable;
+import com.intellij.util.ui.JBUI;
+import java.awt.Component;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import javax.swing.JPanel;
+import javax.swing.JTable;
+import javax.swing.table.AbstractTableModel;
+import javax.swing.table.DefaultTableCellRenderer;
+import javax.swing.table.TableCellRenderer;
+import org.apache.commons.lang.StringUtils;
+import org.sonarlint.intellij.core.ProjectBindingManager;
+import org.sonarlint.intellij.util.SonarLintUtils;
+import org.sonarsource.sonarlint.core.client.api.connected.LoadedAnalyzer;
+
+public class SonarLintProjectAnalyzersPanel {
+  private static final Logger LOGGER = Logger.getInstance(SonarLintProjectAnalyzersPanel.class);
+  private final Project project;
+  private JBTable analyzerTable;
+  private JPanel panel;
+  private JPanel tablePanel;
+  private Model tableModel;
+
+  public SonarLintProjectAnalyzersPanel(Project project) {
+    this.project = project;
+    reload();
+  }
+
+  public JPanel getPanel() {
+    return panel;
+  }
+
+  public void reload() {
+    ProjectBindingManager bindingManager = SonarLintUtils.get(project, ProjectBindingManager.class);
+    try {
+      Collection<LoadedAnalyzer> loadedAnalyzers = bindingManager.getFacade().getLoadedAnalyzers();
+      tableModel.set(loadedAnalyzers);
+    } catch(Exception e) {
+      LOGGER.error(e);
+      tableModel.set(Collections.emptyList());
+    }
+  }
+
+  private void createUIComponents() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "createUIComponents" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUnusedPrivateMethod)